api/server/server.go

```diff
@@ -30,14 +30,14 @@ const versionMatcher = "/v{version:[0-9.]+}"
 
 // Config provides the configuration for the API server
 type Config struct {
-	Logging          bool
-	EnableCors       bool
-	CorsHeaders      string
-	AuthZPluginNames []string
-	Version          string
-	SocketGroup      string
-	TLSConfig        *tls.Config
-	Addrs            []Addr
+	Logging                  bool
+	EnableCors               bool
+	CorsHeaders              string
+	AuthorizationPluginNames []string
```

Comment:
on a sidenote, @calavera don't you think we should name this `AuthorizationPlugins` instead, if we're making this part of the config file eventually? I don't like the `Names` part.